api/src/api/userRouter.test.ts

```diff
@@ -664,8 +661,8 @@ describe("userRouter", () => {
           const result = await request(app).get(`/users/123`).set("Authorization", "Bearer user-123-token");
           const resultBusiness = getCurrentBusiness(result.body as UserData);
           expect(stubTimeStampBusinessSearch.search).toHaveBeenCalled();
-          expect(resultBusiness.formationData.dbaBusinessNameAvailability?.status).toEqual("UNAVAILABLE");
-          expect(resultBusiness.formationData.businessNameAvailability?.status).toEqual("AVAILABLE");
+          expect(resultBusiness.formationData.dbaBusinessNameAvailability?.status).toEqual("AVAILABLE");
+          expect(resultBusiness.formationData.businessNameAvailability?.status).toEqual("UNAVAILABLE");
```

Comment:
stubTimeStampBusinessSearch has status in mockResolvedValue set to UNAVAILABLE, therefore the businessNameAvailability status will update to UNAVAILABLE as well.